dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -282,7 +308,14 @@ def _extract_hunt_results(self, output_file_path):
       print('Output path {0:s} could not be removed: {1:s}'.format(
           output_file_path, exception))
 
-    return collection_paths
+    # Translate GRR client IDs to FQDNs with the information retrieved
+    # earlier
+    fqdn_collection_paths = []
+    for client_id, path in collection_paths:
+      fqdn = client_id_fqdn[client_id]
```

Comment:
Consider handling a KeyError here, otherwise this is going to blow up if we ever get a client with a missing FQDN, or missing client_info. Maybe client_id_fqdn.get(client_id, client_id), so you fail back to the ID if there's a problem.

Reply:
Done.